messages/german.json

```diff
@@ -0,0 +1,1393 @@
+        "death": "Tod",
+        "protection": "Schutz",
+        "silence": "Stille",
+        "revealing": "Aufschlussreich",
+        "desperation": "Verzweiflung",
+        "impatience": "Ungeduld",
+        "pacifism": "Pazifismus",
+        "influence": "beeinflussen",
+        "narcolepsy": "Narkolepsie",
+        "exchange": "Austausch",
+        "lycanthropy": "Lykanthropie",
+        "luck": "Glück",
+        "pestilence": "Pest",
+        "retribution": "Vergeltung",
+        "misdirection": "Fehlleitung",
+        "deceit": "Betrug",
+
+        "hunger": "hunger",
+        "sustenance": "Nahrung"
+    },
+    "_role_categories": {
+        "*": "For assistance in configuring this metadata, please see https://werewolf.chat/Translation#Role_categories",
+        "Wolf": ["Wolf", "Wölfe"],
+        "Wolfchat": ["Wolfchat Mitglied", "Wolfchat Mitglieder"],
+        "Wolfteam": ["Böser Dorfbewohner", "Böse Dorfbewohner"],
+        "Killer": ["Mörder", "Mörder"],
+        "Village": ["Dorfmitglied", "Dorfmitglieder"],
+        "Nocturnal": ["Nächtliche Rolle", "Nächtliche Rollen"],
+        "Neutral": ["Neutrale Spieler", "Neutrale Spieler"],
+        "Win Stealer": ["Siegräuber", "Siegräuber"],
+        "Hidden": ["Versteckte Rolle", "Versteckte Rollen"],
+        "Safe": ["Spezieller Dorfbewohner", "Spezieller Dorfbewohner"],
+        "Spy": ["Spion Rolle", "Spion Rollen"],
+        "Intuitive": ["intuitive Rolle", "intuitive Rollen"],
+        "Cursed": ["Verfluchter Dorfbewohner", "Verfluchte Dorfbewohner"],
+        "Innocent": ["Unschuldiger Spieler", "Unschuldiger Spieler"],
+        "Team Switcher": ["Team Wechsler", "Team Wechsler"]
+    },
+    "_phases": {
+        "*": "For assistance in configuring this metadata, please see https://werewolf.chat/Translation#Phases",
+        "join": "join",
+        "night": "Nacht",
+        "day": "Tag"
+    },
+    "no_victims": [
+        "Die Leiche eines jungen Pinguin-Haustiers wurde gefunden... Wie kann man nur so grausam sein..",
+        "Pfotenabdrücke kreisen um das Dorf.",
+        "Einige Pfotenabdrücke führen zum Dorfplatz.",
+        "Eine Pfütze von Wolfsblut und Pfotenabdrücke wurde gefunden.",
+        "Die Leiche einer getöteten Katze wurde gefunden.",
+        "Einige Haustüren wurden geöffnet.",
+        "Ein paar Häuser haben frisches Blut an den Wänden.",
+        "Ein Duft, ähnlich wie der eines Wolfes liegt in der Luft.",
+        "Halbbegrabene Wolfkots wurden gefunden.",
+        "Spuren von Wolfsfellen wurden gefunden.",
+        "Eine unheimliche Ruhe ist zu spüren.",
+        "Große Kratzspuren befinden sich an einem Baum.",
+        "Die Überreste eines der Dorfschweine wurden gefunden.",
+        "Ein paar Knochen von einem lokalen Eichhörnchen wurden gefunden.",
+        "Der Abdruck eines Wolfsgesichts dessen auf einem Pfahl geritzt wurde befindet sich in der Nähe des Dorfplatzes."
```

Comment:
this one isn't really comprehensible imo

Reply:
I couldn't think of anything better Google translates weird :(

Reply:
i also removed this line 311 i dont found a better translation from the en.json